### ISSUE TITLE
Error handling

### ISSUE THREAD

**Issue by ahalterman** [ahalterman/cloacina#10]
Sometimes something breaks on LexisNexis's end, and it shouldn't clog things up on ours. E.g.:

```
Traceback (most recent call last):
  File "bbc_test_download.py", line 100, in <module>
    pool.map(download_wrapper, sourcelist)
  File "/home/ahalterman/anaconda/lib/python2.7/multiprocessing/pool.py", line 251, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/home/ahalterman/anaconda/lib/python2.7/multiprocessing/pool.py", line 567, in get
    raise self._value
requests.exceptions.ConnectionError: ('Connection aborted.', BadStatusLine("''",))
```

**Comment by ahalterman** [comment]
It's caught now, but only at the highest level. I still need to figure out which function is comes from.

It happens in this range (among others, presumably).
New York Times  2005-05-01  2005-05-09

**Comment by ahalterman** [comment]
There's better error handling now, and this specific error is fixed with 5ff88c4639d4c5ce17984660eb1637ee0d7d554e
